docs/hardware/4-nrf91/2-quickstart/3-flash-firmware/2-bootloader.md

```diff
@@ -0,0 +1,123 @@
+---
+id: bootloader
+title: Flash via USB Bootloader
+---
+
+## Installing Tools
+
+Flashing via USB bootloader requires the `newtmgr` tool.
+
+### Installing Newt Manager (for bootloading)
+
+We recommend you use [Jared Wolff's fork of newtmgr](https://github.com/circuitdojo/mynewt-newtmgr) which includes customization for higher-speed transfers on MCUboot implementations that support it. This tool depends on the Go language so we will also install that.
+
+1. Install the Go Language
+
+    :::note
+    You can install the Go language with `sudo apt install golang-go`, but some package managers have older versions. The following process will install the newest version.
```

Comment:
This gave me permission errors and I had to go back and re-do it as directed below. I say we just direct people to use the steps outlined.

Reply:
Maybe something like "We recommend not to use a package manager (like apt) because of possible permission errors"

Reply:
I changed the note to recommend against using apt. Fixed in https://github.com/golioth/docs/pull/158/commits/f29c818c8b18d518915624383ebe5344242e7dfd#